File: qualifier-week-4/non-graded-assignemts/PPA6 - Not Graded.c
#include <stdio.h>
#include <math.h>

int main() {
    double P, r;
    int t, n;
    scanf("%lf", &P);
    scanf("%lf", &r);

    r /= 100;

    scanf("%d", &t);
    scanf("%d", &n);

    for (int i = 1; i <= t: i++) {
        double A = P * pow(1 + r / n, n * i);
        printf("%.2f\n", A);
    }
    return 0;
}

File: qualifier-week-4/non-graded-assignemts/PPA6 - Not Graded.go
package main

import (
	"fmt"
	"math"
)

func compoundInterest(P, r float64, n, t int) float64 {
	A := P * math.Pow(1+r/float64(n), float64(n*t))
	return A
}

func main() {
	var P, r float64
	var n, t int
	fmt.Scan(&P)
	fmt.Scan(&r)

	r /= 100

	fmt.Scan(&t)
	fmt.Scan(&n)

	for i := 1; i <= t; i++ {
		A := compoundInterest(P, r, n, t)
		println(A)
	}
}
